### DISCUSSION TITLE
How to enable session resumption in TLS1.3?

### DISCUSSION THREAD

**Discussion by hmbb2023** [openssl/openssl#21392] [Q&A]
1. I refer to the method mentioned in this [document](https://stackoverflow.com/questions/75059963/resuming-a-tls-1-3-session-in-openssl), but the solution only support one session. If the system connect with multiple servers, there are multiple sessions, and sessions and servers correspond to each other. That is, when the client wants to resume the session, how to map the URL to the saved sessions?
2. Do I have to implement an update mechanism to update the session list?

**Comment by t8m** [comment]
@mattcaswell could perhaps help
